app/controllers/trains_controller.rb

```diff
@@ -7,12 +7,25 @@ def index
     @trains = TrainApi.new.index
   end
 
-  def edit
-    flash["errors"] = nil
-    @train = TrainApi.new.show(params[:id])
+  def new
+    @train = TrainApi.new.new
```

Comment:
no, its not a request.
the form doesnt need a instantiated api request helper object.
i give you the exact code to use here in one of the comments im pretty sure.

This is the new page for THIS APP. not api. we generally dont talk about other apps inside a different apps PR, so unless explicitly talking about the train api, assume all discussion is about the truck app.

Reply:
> i give you the exact code to use here in one of the comments im pretty sure.

```
def index
  @train = { "active": false }
end
```

is it this one?
Ive tried it but it doesnt fix the `active` radio button issue that i mentioned

Reply:
so setting
`train['active']` and `!train['active']` doesnt work for your checked defaults on the form?

Reply:
> `train['active']` and `!train['active']` doesnt work for your checked defaults on the form?

i think the issue lies with `train["active"]` being saved as a string when its submitted

Reply:
https://stackoverflow.com/questions/19339137/how-to-set-boolean-values-to-radio-buttons-in-rails-4-form

Reply:
> https://stackoverflow.com/questions/19339137/how-to-set-boolean-values-to-radio-buttons-in-rails-4-form

i tried adding the `value` but the boolean still saves as a string

Reply:
```
def index
  @train = { "active": false }
end
```

should be `def new`
since thats the method we are talking about if i put it wrong then its cause i rushed, but we talking about new here.

no matter what though
`@train = TrainApi.new.new` this is wrong, and doesnt do anything